SemiconductorTestLibrary.Extensions/source/InstrumentAbstraction/DCPower/Types.cs

```diff
@@ -150,6 +150,14 @@ public class DCPowerWaveformAcquisitionSettings
         /// The measure trigger type.
         /// </summary>
         public DCPowerMeasureTriggerType MeasureTriggerType { get; set; }
+
+        /// <summary>
+        /// Creates an object to define and set DCPower waveform acquisition specific settings.
+        /// This object is used as a parameter for configuring a DCPower waveform acquisition settings for Pin Site Data <see cref="Measure.ConfigureAndStartWaveformAcquisition(DCPowerSessionsBundle, double, double)"/>
```

Comment:
... for configuring waveform acquisition settings for a DCPower channel.

Reply:
Updated